tests/openwrt/test_status.sh

```diff
@@ -51,22 +54,21 @@ TEST_STATUS=$?
 
 # Collect post-run diagnostics
 echo "Collecting post-test-run diagnostic data..."
-ssh "$TARGET" <<"EOF" >> "${TARGET}_diags.log"
+ssh "$TARGET" <<"EOF" >> "$LOG_DIR/conn_map.txt"
 date
 echo "Output of bml_conn_map:"
 /opt/prplmesh/bin/beerocks_cli -c bml_conn_map 2>&1
 echo "Output of bml_nw_map_query:"
 /opt/prplmesh/bin/beerocks_cli -c bml_nw_map_query 2>&1
-echo "Kernel messages:"
-dmesg
 EOF
 
+ssh "$TARGET" "dmesg" > dmesg.txt
+
 # Capture the logs
 echo "Capturing the prplMesh logs..."
-ssh "$TARGET" "tar -C /tmp/beerocks/logs -czvf - ./" > "${TARGET}_logs.tar.gz"
+scp -r "$TARGET:/tmp/beerocks/logs" > "$LOG_DIR/prplmesh_logs"
```

Comment:
No need for a separate directory IMO, all the log files have distinctive names.
```suggestion
scp -r "$TARGET:/tmp/beerocks/logs"  "$LOG_DIR/"
```

Reply:
I thought it would be easier to read to have it in a separate directory, but it doesn't make a big difference, and for the certification logs we already put them all in the same place so I applied your suggestion.